src/BuildASub.java

```diff
@@ -5,9 +5,18 @@
     public static void main(String[] args) throws Exception {
         String bread = chooseBread();
         String meat = chooseMeat();
+        Boolean salad = addSalad ();
+
+        String saladChoice = "";
+
+        if(salad == true)
+        {
```

Comment:
Curly brackets are down to developers preference, however the team normally makes this consistent :) so far we have put `{` on the same line before nesting the logic. I would try and keep this consistent